frontend/test/metabase/scenarios/question/view.cy.spec.js

```diff
@@ -30,12 +30,20 @@ describe("scenarios > question > view", () => {
       cy.get("@sidebar")
         .contains("Created At")
         .click();
+      cy.findByText("Done").click();
 
-      cy.contains("Count by Created At: Month");
+      cy.contains("Count by Created At: Day");
+
+      // Go back into sidebar
+      cy.contains("Summarize").click();
 
       // change grouping from month to year
+      cy.contains("Summarize by")
+        .parent()
+        .parent()
+        .as("sidebar");
       cy.get("@sidebar")
-        .contains("by month")
+        .contains("by day")
```

Comment:
I think the month -> day change here could be from that temporal fingerprinting bug that was on master. If there's not some other change I'm missing, I think you should rebase onto master.

Reply:
These changes were made on today's master (post bug fix). This change is because of a slight change in frontend design that was merged about a week ago.

Reply:
Do you know what change in design did that? In the UI, I still see "month" as the default:
![image](https://user-images.githubusercontent.com/691495/87325722-d22b9e80-c4ff-11ea-8e4b-c716b42b98fd.png)

Reply:
Hmm... I suppose I assumed it was a change that happened when the `by month` was added on the right... but maybe it was a fluke? Mine's showing up as `by month` again today. I'll keep an eye on it as I continue to run the tests to see if that is something that continues to occationally show up incorrectly.

Reply:
@Opalevanescence "by month" has always been the default, but could it be that you're seeing issue `11439` ?